docs/reference/commandline/dockerd.md

```diff
@@ -17,73 +17,74 @@ keywords: "container, daemon, runtime"
+      --label list                            Set key=value labels to the daemon (default [])
+      --live-restore                          Enable live restore of docker when containers are still running
       --log-driver string                     Default driver for container logs (default "json-file")
   -l, --log-level string                      Set the logging level ("debug", "info", "warn", "error", "fatal") (default "info")
-      --log-opt value                         Default log driver options for containers (default map[])
+      --log-opt map                           Default log driver options for containers (default map[])
       --max-concurrent-downloads int          Set the max concurrent downloads for each pull (default 3)
       --max-concurrent-uploads int            Set the max concurrent uploads for each push (default 5)
-      --metrics-addr string                   Set address and port to serve the metrics api (default "")
+      --metrics-addr string                   Set default address and port to serve the metrics api on
       --mtu int                               Set the containers network MTU
       --oom-score-adjust int                  Set the oom_score_adj for the daemon (default -500)
   -p, --pidfile string                        Path to use for daemon PID file (default "/var/run/docker.pid")
       --raw-logs                              Full timestamps without ANSI coloring
-      --registry-mirror value                 Preferred Docker registry mirror (default [])
-      --seccomp-profile value                 Path to seccomp profile
+      --registry-mirror list                  Preferred Docker registry mirror (default [])
+      --seccomp-profile string                Path to seccomp profile
       --selinux-enabled                       Enable selinux support
-      --shutdown-timeout=15                   Set the shutdown timeout value in seconds
+      --shutdown-timeout int                  Set the default shutdown timeout (default 15)
   -s, --storage-driver string                 Storage driver to use
-      --storage-opt value                     Storage driver options (default [])
+      --storage-opt list                      Storage driver options (default [])
       --swarm-default-advertise-addr string   Set default address or interface for swarm advertised address
       --tls                                   Use TLS; implied by --tlsverify
-      --tlscacert string                      Trust certs signed only by this CA (default "/root/.docker/ca.pem")
-      --tlscert string                        Path to TLS certificate file (default "/root/.docker/cert.pem")
-      --tlskey string                         Path to TLS key file (default "/root/.docker/key.pem")
+      --tlscacert string                      Trust certs signed only by this CA (default "/home/fedora/.docker/ca.pem")
```

Comment:
I just pasted the output of `--help` as I thought had been done in the past to generate this content. I assume it might differ depending on the distribution you run it on, which means I don't know what distribution we should assume in the help output. This is another reason I think it's kind of dumb to have to maintain this output manually.